src/components/ContributorsSection.test.tsx

```diff
@@ -5,35 +5,74 @@ import { ContributorsSection } from "./ContributorsSection";
 
 describe("ContributorsSection", () => {
   it("renders a 'Contributors' heading", async () => {
-    const pkg = {
-      name: "package-name",
-      repository: "github:dbartholomae/test-repo",
-    };
-
     expect(
-      await render(<ContributorsSection title="Contributors" pkg={pkg} />)
+      await render(
+        <ContributorsSection
+          contributors={[]}
+          title="Contributors"
+          owner="dbartholomae"
+          repo="test-repo"
+        />
+      )
     ).toContain("## Contributors");
 
     expect(
-      await render(<ContributorsSection pkg={pkg} />)
+      await render(
+        <ContributorsSection
+          contributors={[]}
+          owner="dbartholomae"
+          repo="test-repo"
+        />
+      )
     ).toContain("## ‍💼 Contributors");
   });
 
   it("renders sentence with a link to the contributors on GitHub", async () => {
-    const pkg = {
-      name: "package-name",
-      repository: "github:dbartholomae/test-repo",
-    };
-    expect(await render(<ContributorsSection pkg={pkg} />)).toContain(
+    expect(
+      await render(
+        <ContributorsSection
+          contributors={[]}
+          owner="dbartholomae"
+          repo="test-repo"
+        />
+      )
+    ).toContain(
       "This package only works thanks to [all of our contributors](https://github.com/dbartholomae/test-repo/graphs/contributors)"
     );
   });
 
-  it("renders nothing when repository is not hosted on GitHub", async () => {
-    const pkg = {
-      name: "package-name",
-      repository: "bitbucket:user/test-repo",
-    };
-    expect(await render(<ContributorsSection pkg={pkg} title="Contributors" />)).toBe("")
+  it("renders a list of contributors on GitHub", async () => {
+    const contributors = [
+      {
+        id: 1,
+        name: "dbartholomae",
+      },
+    ];
+    expect(
+      await render(
+        <ContributorsSection
+          contributors={contributors}
+          owner="dbartholomae"
+          repo="test-repo"
+        />
+      )
+    ).toContain(
+      '[![dbartholomae](https://avatars.githubusercontent.com/u/1?size=100 "dbartholomae")](https://github.com/dbartholomae "dbartholomae\'s GitHub profile\'")'
+    );
+  });
+
+  it("renders a show more contributors link when additional contributors count is provided", async () => {
+    expect(
+      await render(
+        <ContributorsSection
+          contributors={[]}
+          owner="dbartholomae"
+          repo="test-repo"
+          additionalContributorsCount={1}
+        />
+      )
+    ).toContain(
+      "[+ 1 contributors](https://github.com/dbartholomae/test-repo/graphs/contributors)"
```

Comment:
Please add a test for what the main section looks like with an empty contributors array. In that case, it could make sense to include the number of contributors directly in the first sentence instead of doubling the link.
I know it's an edge case, but it should be easy enough to fix :)